sdk/cosmos/azure-cosmos/README.md

```diff
@@ -952,7 +952,7 @@ database = client.create_database(DATABASE_NAME, logger=logger)
 ```
 **NOTICE: The Following is a Preview Feature that is subject to significant change.**
 To further customize what gets logged, you can use a  **PREVIEW** diagnostics handler to filter out the logs you don't want to see.
-There are several ways to use the diagnostics handler, those include the following:
+There are several ways to use the diagnostics handler, you can take a look at the samples [here][cosmos_diagnostics_handler_sample] or take a look at the following:
```

Comment:
Why is the "diagnostic handler" still in the API?  The logging filters were supposed replace this using built-in Pythonic concepts?

Reply:
Through some in-depth analysis of the performance of the two approaches, I witnessed that the "diagnostic handler" approach had a performative edge over just using filters, however there is definitely an advantage to using filters especially if we can implement a centralized logger for the sdk where we can make it easy to filter all logs coming from the SDK in general, vs the diagnostic handler being specific to just the HTTP Resquests/response, but I do believe having that performance improvement is beneficial and has a use case to customers since the performance difference is more apparent with larger volumes of requests. Both are entirely optional as well.